julia-mode.el

```diff
@@ -301,6 +297,21 @@
 (defconst julia-block-end-keywords
   (list "end" "else" "elseif" "catch" "finally"))
 
+(defsubst julia-syntax-comment-or-string-p (&optional syntax-ppss)
```

Comment:
are we moving this function from lower below to up here to group similar functions together?

on the matter, this looks like an internal function. maybe next time we'll rename these kind of functions as `julia--…`

Reply:
`defsubst` invocations need to be before their invocation for the elisp to byte-compile cleanly. I think I moved the other two to be close to this one while I was at it.

Reply:
It would be great if you could open an issue listing these functions, so that we don't forget.